graphql/README.md

```diff
@@ -0,0 +1,46 @@
+# Scalar DB GraphQL Server
+
+## Build & Install
+
+To build and install the Scalar DB GraphQL Server, use gradle installDist, which will build the source files and install an executable, required jars:
+
+```
+$ ./gradlew installDist
+```
+
+## Run
+
+This runs Scalar DB GraphQL Server:
+
+```
+$ cd graphql/build/install/graphql
+$ export SCALARDB_GRAPHQL_SERVER_OPTS="<your JVM options>"
+$ bin/scalardb-graphql-server --config <your configuration file path>
```

Comment:
Just a question. `your configuration file` is a Scalar DB properties file?
If so, it is probably better to explicitly say so.

One more question.
Nothing needs to be configured to start using graphql?

Reply:
> your configuration file is a Scalar DB properties file?

Yes.

I basically copied the scalardb-server's [`README.md`](https://github.com/scalar-labs/scalardb/blob/2044bcca5a25ffdb41efc91073d3fd9577eafdf3/server/README.md). But the scalardb-server has also [`scalardb-server.md`](https://github.com/scalar-labs/scalardb/blob/2044bcca5a25ffdb41efc91073d3fd9577eafdf3/docs/scalardb-server.md) in the `docs` directory. Both contain information about installing and running the server, but with different levels of detail.
Is it probably good the build instructions go to README.md, and the user documentation such as running and properties go to the docs directory? Cc: @brfrn169 

> Nothing needs to be configured to start using graphql?

At least the `scalar.db.graphql.namespace` property is required since we need at least one table that the GraphQL layer can communicate with. I will write for this in the user documentation later in the right place.

Reply:
> I basically copied the scalardb-server's README.md. But the scalardb-server has also scalardb-server.md in the docs directory. Both contain information about installing and running the server, but with different levels of detail.
> Is it probably good the build instructions go to README.md, and the user documentation such as running and properties go to the docs directory?

Thank you for pointing it out! Yes, we have two documents for Scalar DB server, and it looks like we need to re-arrange them as you mentioned. I'll do that when I get a chance. Thanks.

Reply:
Later I'd like to add a user doc in `docs` directory, for now I have updated the `graphql/README.md`. Take a look, please!